lib/compilers/fake-for-test.js

```diff
@@ -30,11 +30,10 @@ export class FakeCompiler {
     }
 
     constructor(info) {
-        this.compiler = {
-            id: info.id || 'fake-for-test',
-            lang: info.lang || 'fake-lang',
-            options: info.options || '',
-        };
+        this.compiler = Object.assign({}, info);
+        this.compiler.id = info.id || 'fake-for-test';
```

Comment:
I _think_ you can do this as
`this.compiler = Object.assign({id:'fake-for-test', lang:'fake-lang', options:''}, info);` (if I remember right? the assign from the right param overwrites the left?